common/djangoapps/status/models.py

```diff
@@ -0,0 +1,60 @@
+"""
+Store status messages in the database.
+"""
+
+from django.db import models
+from django.contrib import admin
+from django.core.cache import cache
+
+from xmodule_django.models import CourseKeyField
+
+from config_models.models import ConfigurationModel
+from config_models.admin import ConfigurationModelAdmin
+
+
+class GlobalStatusMessage(ConfigurationModel):
+    """
+    Model that represents the current status message.
+    """
+    message = models.TextField(blank=True, null=True)
+
+    def full_message(self, course_key):
+        """ Returns the full status message, including any course-specific status messages. """
+        cache_key = "status_message.{course_id}".format(course_id=unicode(course_key))
```

Comment:
Can you explain which cache you are using here. Is this cached for everybody for a certain duration? I would think that we could cache the message for at least five minutes.